base/array.jl

```diff
@@ -791,9 +791,9 @@ function find(testf::Function, A::AbstractArray)
     I
 end
 
-function find(A::AbstractArray)
+function find(A)
     nnzA = countnz(A)
-    I = similar(A, Int, nnzA)
+    I = Vector{Int}(nnzA)
```

Comment:
Yeah these methods with size arguments to `similar` always make me scratch my head in terms of what they're for. I guess `similar(::SparseVector, T, n)` could return something other than `Vector`, but wouldn't make too much sense to use that for `find`.